src/main/java/uk/gov/hmcts/reform/iahomeofficemockapi/generated/infrastructure/api/ApplicationStatusApi.java

```diff
@@ -148,7 +148,8 @@ private String getResponseJson(String homeOfficeReference) throws IOException {
                 responseJsonFile = "ho_api_get-by-search-params_200_no-data-found-uan-ref.json";
                 break;
             default:
-                throw new IOException("Invalid home office reference : " + homeOfficeReference);
+                responseJsonFile = "ho-api_get-by-serach-params_200_valid-response-default.json";
```

Comment:
Earlier, we had discussed and you informed that we should differentiate what is valid data in the PRP1 and something that is not found. Will we go with this approach, which I prefer, as it hinders testing for any other HO reference

Reply:
THat's right. That is the reason behind sending back a default response. For anyone else using this need not have to know the test ids we are interested to use in testing